.circleci/config.yml

```diff
@@ -175,18 +175,74 @@ jobs:
       - run: make -C build -j4
       - run: make -C build -j2 tests
       - run: make -C build -j2 check
-      - run: make -C build install
+      - save_cache:
+          key: ccache
+          paths:
+            - ~/.ccache
+
+  build-release-binary-osx:
+    macos:
+      xcode: "9.2.0"
+    environment:
+      HOMEBREW_NO_AUTO_UPDATE: 1
+    steps:
+      - run: brew install protobuf --without-python@2
+      - run: brew upgrade python
+      - run: brew install cmake libtool boost-python libspatialite pkg-config lua jq curl wget czmq lz4 nvm
+      - run:
+          command: |
+            set +e
+            touch $BASH_ENV
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.5/install.sh | bash
+            echo 'export NVM_DIR="$HOME/.nvm"' >> $BASH_ENV
+            echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
+            echo 'nvm install v8.11.2' >> $BASH_ENV
+            echo 'nvm alias default v8.11.2' >> $BASH_ENV
+      - checkout
+      - run: git submodule sync && git submodule update --init
+      - run: npm install --ignore-scripts
+      - run: mkdir -p build
+      - run: cd build && cmake .. -DBUILD_SHARED_LIBS=Off -DENABLE_PYTHON_BINDINGS=Off -DENABLE_DATA_TOOLS=OFF -DENABLE_SERVICES=OFF -DBoost_USE_STATIC_LIBS=ON -DProtobuf_USE_STATIC_LIBS=ON -DLZ4_USE_STATIC_LIBS=ON
+      - run: make -C build -j4
       - run:
           command: |
             export PACKAGE_JSON_VERSION=$(node -e "console.log(require('./package.json').version)")
             export PUBLISH=$([[ "${CIRCLE_TAG:-}" == "v${PACKAGE_JSON_VERSION}" ]] && echo "On" || echo "Off")
             export COMMIT_MESSAGE=$(git log --oneline --format=%B -n 1 ${CIRCLE_SHA1} | head -n 1)
             export BUILD_TYPE=Release
             ./scripts/publish.sh
-      - save_cache:
-          key: ccache
-          paths:
-            - ~/.ccache
+
+  build-debug-binary-osx:
```

Comment:
I think we can use CircleCI's git tag option to only run these jobs on tagged commits https://circleci.com/docs/2.0/workflows/#git-tag-job-execution